doc/whats_new/v0.23.rst

```diff
@@ -154,10 +154,14 @@ Changelog
 
 - |Enhancement| :func:`gaussian_process.kernels.Matern` returns the RBF kernel when ``nu=np.inf``.
   :pr:`15503` by :user:`Sam Dixon <sam-dixon>`.
-  
+
 :mod:`sklearn.linear_model`
 ...........................
 
+- |Feature| Support of `sample_weight` in :class:`linear_model.ElasticNet` and
+  :class:`linear_model:Lasso` for dense feature matrix `X`.
```

Comment:
you don't have it for the CV estimators?

Reply:
No. The what's new is correct. Sample weight support is added only to `ElasticNet` and `Lasso`. Unfortunately, the logic in `_coordinate_descent.py` is such that this feature is not automatically inherited by `ElasticNetCV` and others. I do not know if there are advantages with the current structure.
In any case, I'd prefer to put adding sample weights to ElasticNetCV in another PR. What do you think?